acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/privacy/PrivTraceTransactionAcceptanceTest.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.privacy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.web3j.utils.Restriction.UNRESTRICTED;
+
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.tests.acceptance.dsl.privacy.ParameterizedEnclaveTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.privacy.PrivacyNode;
+import org.hyperledger.besu.tests.acceptance.dsl.privacy.account.PrivacyAccountResolver;
+import org.hyperledger.besu.tests.web3j.generated.SimpleStorage;
+import org.hyperledger.enclave.testutil.EnclaveEncryptorType;
+import org.hyperledger.enclave.testutil.EnclaveType;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.Optional;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.web3j.utils.Restriction;
+
+public class PrivTraceTransactionAcceptanceTest extends ParameterizedEnclaveTestBase {
+
+  private final PrivacyNode node;
+
+  private final PrivacyNode wrongNode;
+
+  public PrivTraceTransactionAcceptanceTest(
+      final Restriction restriction,
+      final EnclaveType enclaveType,
+      final EnclaveEncryptorType enclaveEncryptorType)
+      throws IOException {
+
+    super(restriction, enclaveType, enclaveEncryptorType);
+
+    node =
+        privacyBesu.createPrivateTransactionEnabledMinerNode(
+            restriction + "-node",
+            PrivacyAccountResolver.ALICE.resolve(enclaveEncryptorType),
+            enclaveType,
+            Optional.empty(),
+            false,
+            false,
+            restriction == UNRESTRICTED);
+
+    wrongNode =
+        privacyBesu.createPrivateTransactionEnabledMinerNode(
+            restriction + "-node",
+            PrivacyAccountResolver.BOB.resolve(enclaveEncryptorType),
+            enclaveType,
+            Optional.empty(),
+            false,
+            false,
+            restriction == UNRESTRICTED);
+
+    privacyCluster.start(node);
+    privacyCluster.start(wrongNode);
+  }
+
+  @Test
+  public void getTransactionTrace() throws JsonProcessingException {
```

Comment:
Could you add a test verifying that in multi tenancy you can only access your own privacy groups.

Reply:
Hey @pinges, I looked into current acceptance tests (also in privacy/multitenancy tests) but I couldn't find a similar verification test for any priv RPC API. Can you point me to a similar test to which I can refer?

Reply:
I think actually having another look, the createWrongPrivacyGroup() method call covers this case